### ISSUE TITLE
Accessibility Support on Create BreakRoom 

### ISSUE THREAD

**Issue by galupa** [bigbluebutton/bigbluebutton#9316]
**Is your feature request related to a problem? Please describe.**
Currently on create break room page, the user selection is using drag and drop. 
If possible it will be good if there support for keyboard users (user who cannot see / use mouse).

**Describe the solution you'd like**
Maybe create the user list to be tab-able (using tabindex) and add some button or options to assign on which room.

**Describe alternatives you've considered**
Other solution should be ok, as long as the keyboard user able to assign user to each room.

**Additional context**
Maybe something like this will be useful (take from bluejean)
<img width="207" alt="Untitled" src="https://user-images.githubusercontent.com/29434066/80563222-87d96c00-8a3e-11ea-9c6f-cae2d4ca70dd.png">

**Comment by antobinary** [comment]
Implemented on 2.3 via https://github.com/bigbluebutton/bigbluebutton/pull/12078
